fcn_segmentation_node:
  ros__parameters:
    engine_file: "fcn_resnet50_1238x374.trt"
    width: 1238
    height: 374
    num_classes: 21
